README.md

```diff
@@ -23,8 +23,8 @@ housekeeping:
   schema-name: my_db
   # Connection details
   data-source:
-    # The package of your driver class
-    driver-class-name: com.mysql.cj.jdbc.Driver
+    # The package of your driver class; note that MySQL on AWS needs a MariaDB driver
```

Comment:
Aurora RDS which can be MySQL/Postgress flavoured needs MariaDB. If you use the MySQL RDS instance you probably still need the MySQL driver.

Maybe something like:
Note some users have seen issues using the MySQL driver on AWS Aurora, [AWS recommends](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Connecting.html) to use the MariaDB driver for those use cases.

Reply:
I will add it at the _Notes_ section, because the link can't get included in a code section :)

Reply:
I think we should just leave the text here as "The name of your JDBC Driver class" and don't say anything more, but have the mariadb driver as the example.

Reply:
Done :)